pykickstart/commands/url.py

```diff
@@ -241,3 +241,57 @@ def _getParser(self):
                         The metalink URL to install from. Variable substitution
                         is done for $releasever and $basearch in the url.""")
         return op
+
+class F30_Url(F27_Url):
+    removedKeywords = F27_Url.removedKeywords
+    removedAttrs = F27_Url.removedAttrs
+
+    def __init__(self, *args, **kwargs):
+        F27_Url.__init__(self, *args, **kwargs)
+        self.sslcacert = kwargs.get("sslcacert", None)
+        self.sslclientcert = kwargs.get("sslclientcert", None)
+        self.sslclientkey = kwargs.get("sslclientkey", None)
+
+    def __str__(self):
+        retval = KickstartCommand.__str__(self)
```

Comment:
I didn't do that, because `F27_Url.__str__(self)` adds a newline. I simply copied what the older classes were doing. I think removing the newline before adding the other options is a bit weird - what do you think?

Reply:
Yes, it is weird, but it is a common practice in pykickstart. Look at other commands, for example:
https://github.com/dcantrell/pykickstart/blob/master/pykickstart/commands/clearpart.py#L162

Reply:
Interesting. Thanks for the pointer. I've pushed a change.